.github/workflows/pr-auth-dependabot.yml

```diff
@@ -0,0 +1,26 @@
+name: auth dependabot myself
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  dependabot-comments:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: "14.15.4"
+      - run: yarn install
+      - name: "Writing comments"
+        uses: actions/github-script@v4
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            const path = require('path');
+            const scriptPath = path.resolve('./.github/workflows/auth-writing-comments.js');
+            const { getOctokit } = require(`${process.env.GITHUB_WORKSPACE}/node_modules/@actions/github`)
```

Comment:
Learned: import module in `github-script`, [ref](https://github.com/actions/github-script/issues/73#issuecomment-675469777)

Reply:
You can also import modules in the external script, see #58.